src/lxml/parser.pxi

```diff
@@ -947,6 +981,10 @@ cdef class _BaseParser:
         return 0
 
     cdef xmlparser.xmlParserCtxt* _newParserCtxt(self) except NULL:
+        """
+        This method calls into libxml to configure the libxml2-level parser context,
+        among other things.
+        """
```

Comment:
```suggestion
        """
        Create a libxml2-level parser context.
        """
```

Reply:
The reason I wrote `among other things` is that the lines
`                self._registerHtmlErrorHandler(c_ctxt)`
and
`        c_ctxt.sax.startDocument = _initSaxDocument`
are important and not related to creating the parser context. This is especially true with the line about SAX, which does many things.

Reply:
Well, there is a libxml2 function to create the context. If the creation was trivial, we wouldn't need this function here to create it.
```suggestion
        """
        Create and initialise a libxml2-level parser context.
        """
```
Better?

Reply:
I would advocate getting rid of the docstring entirely and replace it with comments in the right places. We could avoid the words "among other things" by just not putting a comment above the lines with "other things" (for example `c_ctxt.sax.startDocument = _initSaxDocument`).